contrib/kind.yaml.j2

```diff
@@ -0,0 +1,76 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  # the default CNI will not be installed
+  disableDefaultCNI: true
+  apiServerAddress: {{ ovn_apiServerAddress | default('11.12.13.1') }}
+  apiServerPort: 11337
+{%- if ovn_ip_family %}
+  ipFamily: {{ ovn_ip_family }}
+{%- endif %}
+kubeadmConfigPatches:
+- |
+{%- if ovn_ip_family is equalto "DualStack" %}
```

Comment:
I've updated my KIND fork with dual stack, right now you only need to configure `ipFamily: DualStack` and it automatically configures the feature_gates in the cluster

Reply:
Antonio, thanks for taking the time to look over. Do you have any plans to update https://github.com/aojea/kind/releases/download/dualstack/kind.linux with your latest changes? I pulled in your PR, built locally and works as you described, but this image is still based on 4/19 version of the PR. I just need to update my documentation (docs/kind-ipv6-dualstack.md) if you don't plan on keeping it updated.

Reply:
sure, I think I can have time at the end of the week, will keep you posted

Reply:
@Billy99 https://github.com/aojea/kind/releases/tag/dualstack

I've updated the branch to avoid patching kubeadm manually, right now it just needs

```yaml
# a dual-stack cluster
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
networking:
  ipFamily: DualStack
nodes:
- role: control-plane
- role: worker
- role: worker
```